black.py

```diff
@@ -188,12 +188,23 @@ class Feature(Enum):
 }
 
 
-@dataclass
 class FileMode:
```

Comment:
Is dataclass support planned for mypyc?

Reply:
There's a bug open for it (https://github.com/mypyc/mypyc/issues/671) but I'm not sure when I'll have a chance to do it

Reply:
I think I'm going to have dataclass support without optimized `__init__` in the next few days, so we'll be able to keep using dataclasses though I'll probably keep the `__init__` implementation on `Line` and anything else that gets created a lot.